utils/issue-format-bot/lib/issueedit.js

```diff
@@ -20,6 +20,13 @@ module.exports = function(robot, alexa) {
       return;
     }
 
+    const createdAt = new Date(context.payload.issue.created_at);
+    // September 25th, 2017
+    if (createdAt.getFullYear() <= 2017 && createdAt.getMonth() <= 8 && createdAt.getDate() <= 25) {
```

Comment:
```js
const botCreatedDate = new Date(2017, 10, 25)
if (createdAt < botCreatedDate) {
```

Reply:
Nice, thanks! I had no idea you could compare `Date`s like this; I was gonna compare Unix timestamps.